src/main/resources/db/migration/V2_5__emplyment_wa_flag_config_tables.sql

```diff
@@ -0,0 +1,8 @@
+-- insert iac base flag into flag_config table
+INSERT INTO flag_config (flag_name, env, service_name, status) VALUES ('employment_wa_1_0', 'local', 'sscs', 'true');
```

Comment:
Not require any flags at case level. only control require at the level of WA roles.
reason: Case roles will be created on top of ORG roles.